tunnel/IpTunnel.h

```diff
@@ -36,11 +36,21 @@ struct IpTunnel_Connection
     /** The IPv4 address used for this connection or all zeros if none was assigned. */
     uint8_t connectionIp4[4];
 
-    /** The IPv6 netmask/prefix length, in bits. Defaults to 128 if none was assigned. */
-    uint8_t connectionIp6Prefix;
+    /** The IPv6 netmask/prefix length from server, in bits.
+     * Defaults to 128 if none was assigned. */
+    uint8_t connectionIp6RoutedPrefix;
```

Comment:
Maybe I don't understand properly because as far as I can tell, it will route stuff into the IPTunnel as long as it has 1 destination IP which is quite useless, the most common is to route all ipv6 traffic into the tunnel.